### DISCUSSION TITLE
Resource blocked due to MIME type mismatch (X-Content-Type-Options: nosniff)

### DISCUSSION THREAD

**Discussion by Thovi98** [photoprism/photoprism#3656] [Q&A]
Hello!
I’m trying to setup PhotoPrism with the unofficial builds [frome this respository](https://github.com/IndrekHaav/photoprism-builds). Installation goes great, but when I try to open PhotoPrism, I come upon this page:
![261807229-022c2355-e4ee-462b-82f7-2332f9665523](https://github.com/photoprism/photoprism/assets/51749973/3ed55e26-ee4f-4fdf-9801-f84533d1d73f)

When I open my browser console, I see this:
![261807390-962de974-b776-4ee7-adcd-7dc86b07f1e0](https://github.com/photoprism/photoprism/assets/51749973/27a7711a-b404-49e6-b081-1e206b640680)

No privacy extensions are enabled when I try to open it.

I would like to ask you if you have some ideas of the problem here? A misconfigured nginx configuration (although I’ve copied the one from PhotoPrism docs)? A problem with the Photoprism configuration file?

I would like to install those unofficial builds to bypass my problem to run PhotoPrism on Bullseye (see my precedent message here: https://github.com/photoprism/photoprism/discussions/3123).

If someone has an idea to solve this issue I would be very glad! Thanks :D

**Comment by lastzero** [comment]
Looks like you just built the backend but not the frontend?

**Reply by Thovi98** [reply]
Thanks for your answer! I don’t think so, because I’ve the `/assets` folder where I can see my css and js files (in /assets/static/build)?

**Reply by lastzero** [reply]
https://docs.photoprism.app/developer-guide/setup/

**Reply by Thovi98** [reply]
The purpose of https://github.com/IndrekHaav/photoprism-builds is to build the backend and the frontend. I’ve compared my files with a working PhotoPrism instance (working with Docker Extract), and it looks similar anyway. Normally, after extracting the unofficial build, I just have to configure PhotoPrism but it looks there’s an issue. Maybe it’s in the build process (although it works for the repository maintainer).

**Reply by Thovi98** [reply]
Anyway, if you haven’t more clue about the issue here, it’s not a problem. I wanted to try to update the PhotoPrism package on Yunohost (because the actual version is rather obsolete… I have seen some security fixes since 1 year…). I don’t want to bother you too much about this :)
You make a really good job with this app!
